ballerina/file_csv_io.bal

```diff
@@ -13,6 +13,18 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
+import ballerina/jballerina.java;
+
+// # Read file content as a CSV.
+// # ```ballerina
+// # string[][]|io:Error content = io:fileReadCsv("./resources/myfile.csv");
+// # ```
+// # + path - The CSV file path
+// # + skipHeaders - Number of headers, which should be skipped prior to reading records
+// # + return - The entire CSV content in the channel as an array of string arrays or an `io:Error`
+// public isolated function fileReadCsv(string path, int skipHeaders = 0) returns string[][]|Error {
+//     return channelReadCsv(check openReadableCsvFile(path, COMMA, DEFAULT_ENCODING, skipHeaders));
+// }
```

Comment:
Don't comment the code. you can do the change in the same API definition. no need to comment and add a new one.

Reply:
Done, they were previously left commented considering the future changes in runtime.